pkg/relayer/proxy/sync.go

```diff
@@ -189,13 +201,22 @@ func (server *relayMinerHTTPServer) serveSyncRequest(
 	}
 
 	// Send the relay request to the native service.
+	serviceCallStartTime := time.Now()
 	httpResponse, err := client.Do(httpRequest)
 	if err != nil {
 		// Do not expose connection errors with the backend service to the client.
+		// Capture the service call request duration metric.
+		relayer.CaptureServiceDuration(serviceId, serviceCallStartTime, 500)
```

Comment:
Use `statusCode` variable declared earlier

```suggestion
		relayer.CaptureServiceDuration(serviceId, serviceCallStartTime, statusCode)
```